apps/catalogue/src/views/HarmonizationRow.vue

```diff
@@ -42,10 +42,11 @@ export default {
       }, {});
     },
     getMatchStatus(resource) {
-      if (this.variable.repeats) {
+      if (this.variable.repeats && Array.isArray(this.resourceMappings)) {
```

Comment:
how could resourceMappings not ben an array ? ( use ts ?)

Reply:
Please remove this and then you will see errors in the console. My hypothesis is than it returns null if no mappings are available

Reply:
suggest .?

Reply:
done